job-dsls/src/main/resources/job-scripts/prod_osl_trigger_umb_message.jenkinsfile

```diff
@@ -53,10 +55,20 @@ pipeline {
                 }
             }
         }
+        stage('Fetch images tags') {
```

Comment:
I have 2 suggestions here:
1) Move this part that gets the image registry information to a method

2) We could do the same as we do for the PRODUCT_ID and MILESTONE and makes it as option parameter in the job, so in case it is not used we could take the latest available information, in that case we would take the latest available image. You could do it with:

$ brew list-builds --package=openshift-serverless-1-logic-data-index-ephemeral-rhel8-container | tail -n1 |  cut -d " " -f1

It will return `openshift-serverless-1-logic-data-index-ephemeral-rhel8-container-1.25.0-4`

Then you can use:
$ brew buildinfo openshift-serverless-1-logic-data-index-ephemeral-rhel8-container-1.25.0-4 

And btw, we need to check if `brew` command is available in the Jenkins slaves machines.

Reply:
@rgdoliveira I agree on your proposal, in this way we don't force to input the brew build id.

> And btw, we need to check if brew command is available in the Jenkins slaves machines.

Yes it is, I already tried here: https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/custom/job/alampare/job/PROD/job/osl-trigger-umb-message-test/27

Reply:
Let me update my second suggestion in previous comment. It will not work quite well when there is versions like 1.25.0-1, 1.25.0-10 and 1.25.0-2, as it will sort by string.

So we could try this one
$ brew  latest-build <tag> <package>

and it will become
$ brew  latest-build openshift-serverless-1.25-rhel-8-container-candidate openshift-serverless-1-logic-data-index-ephemeral-rhel8-container

the `1.25` tag version can be derived from the MILESTONE variable